### PR TITLE
Minor tweaks

### PR DESCRIPTION
**Pull request by omarroth**
Minor correction to definition of CFGs. Minor changes to several sentences to (hopefully) help them read more clearly.

### CODE REVIEW

File: recce.ltx [hunk 1]
```diff
@@ -2303,7 +2302,7 @@ A \dfn{context free grammar} (CFG) is a 4-tuple,
 \begin{equation}
 \begin{gathered}
 \text{$[\Vsymset{nt}, \Vsymset{term}, \Vruleset{rules}, \Vsym{accept}]$ such that} \\
-\Vsymset{nt} \cup \Vsymset{term} = \emptyset, \\
```
Comment (omarroth) [inline recce.ltx:2306]: This would be a trivial grammar, where terminals and non-terminals are both the empty set :)

**Comment by jeffreykegler** [comment]
Thanks for these!

**Comment by jeffreykegler** [comment]
@omarroth  Merged the whole thing.  Some of the style tweaks I prefer as is, and I'll revert them, but a long list full of valid changes, some of them catching embarrassing errors!  Thanks!

File: recce.ltx (hunk 1 above)
Comment (jeffreykegler) [reply recce.ltx:2306]: It would have made a lot of my theorems vacuously true, shortening the proofs considerably. :-)

**Pull request merged**